<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FuncTion</class>
 <widget class="QWidget" name="FuncTion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>1000</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/Functionpic/back.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>123</width>
     <height>145</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>700</y>
     <width>1200</width>
     <height>300</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>770</y>
     <width>1011</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255,0.3);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>770</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255,0.3);
font: 75 italic 14pt &quot;Comic Sans MS&quot;;
border-width:0;
border-style:outset;
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>770</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255,0.3);
font: 75 italic 14pt &quot;Comic Sans MS&quot;;
border-width:0;
border-style:outset;
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>1000</width>
     <height>721</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255,0);
border-color: rgb(255, 255, 255,0);
</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>701</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255,0.3);
border-width:0;
border-style:outset;
border-radius: 20px;
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="find">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>0</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #ffffff;
    border: 1px solid #dcdfe6;
    padding: 10px;
    border-radius: 5px;
}

QPushButton:hover {
	background-color: #caffb9;
    color: #409eff;
}

QPushButton:pressed, QPushButton:checked {
	background-color: rgb(255, 255, 255,0.3);
    border: 1px solid #3a8ee6;
    color: #409eff;
}
</string>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QPushButton" name="likesongs">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 16pt &quot;Dubai Medium&quot;;
text-decoration: underline;
background-color: rgb(255, 255, 255,0);
border-color: rgb(255, 116, 23);
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string>喜欢</string>
   </property>
  </widget>
  <widget class="QPushButton" name="colorselect1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 16pt &quot;Dubai Medium&quot;;
text-decoration: underline;
background-color: rgb(255, 255, 255,0);
border-color: rgb(255, 116, 23);
color: rgb(255, 224, 151);
</string>
   </property>
   <property name="text">
    <string>明星</string>
   </property>
  </widget>
  <widget class="QPushButton" name="colorselect2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 16pt &quot;Dubai Medium&quot;;
text-decoration: underline;
background-color: rgb(255, 255, 255,0);
border-color: rgb(255, 116, 23);
color: rgb(255, 170, 255);
</string>
   </property>
   <property name="text">
    <string>粉色</string>
   </property>
  </widget>
  <widget class="QPushButton" name="colorselect3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>440</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 16pt &quot;Dubai Medium&quot;;
text-decoration: underline;
background-color: rgb(255, 255, 255,0);
border-color: rgb(255, 116, 23);
color: rgb(14, 26, 255);
</string>
   </property>
   <property name="text">
    <string>经典</string>
   </property>
  </widget>
  <widget class="QPushButton" name="colorselect4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>530</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 16pt &quot;Dubai Medium&quot;;
text-decoration: underline;
background-color: rgb(255, 255, 255,0);
border-color: rgb(255, 116, 23);
color: rgb(120, 255, 129);
</string>
   </property>
   <property name="text">
    <string>动漫</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>200</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>查看天气</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>810</y>
     <width>1200</width>
     <height>191</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="picture">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="singername">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255,0.3);
text-decoration: underline;
font: 14pt &quot;Arial&quot;;
color: rgb(255, 170, 0);
border-width:0;
border-style:outset;
border-radius: 20px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="songname">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255,0.3);
border-width:0;
border-style:outset;
border-radius: 20px;
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="frontsong">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>70</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255,0.3);
image: url(:/Functionpic/frontsong.png);
	 min-width:  48px;
	max-width:  48px;
    min-height: 48px;
    max-height: 48px;
    border-radius: 24px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pause">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>70</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255,0.3);
background-image: url(:/Functionpic/start.png);
	 min-width:  48px;
	max-width:  48px;
    min-height: 48px;
    max-height: 48px;

    border-radius: 24px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="nextsong">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>70</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255,0.3);
image: url(:/Functionpic/nextsongs.png);
	 min-width:  48px;
	max-width:  48px;
    min-height: 48px;
    max-height: 48px;
    border-radius: 24px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>label_3</zorder>
  <zorder>horizontalSlider</zorder>
  <zorder>label_6</zorder>
  <zorder>label_5</zorder>
  <zorder>lineEdit</zorder>
  <zorder>find</zorder>
  <zorder>likesongs</zorder>
  <zorder>tableView</zorder>
  <zorder>colorselect1</zorder>
  <zorder>colorselect2</zorder>
  <zorder>colorselect3</zorder>
  <zorder>colorselect4</zorder>
  <zorder>pushButton</zorder>
  <zorder>frame</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
